main.css

```diff
@@ -0,0 +1,46 @@
+/*
+
+orange: #f5c85f
+gray-bg: #EEEEEE
+dark-gray text: #111111
+gray text: #999999
+
+Font: Open Sans
+
+*/
+
+html {
+  font-family: Open Sans;
+}
+
+body {
+  min-height: 100%;
+  margin: 0;
+  padding-top: 30px;
+
+  /* Centrado de elementos */
+  display: flex;
```

Comment:
I would create a div that wraps all the elements in this section and apply the style to that div and not to the whole body. So that when you create the other sections this style is not automatically applied.